request/builder.js

```diff
@@ -328,20 +345,25 @@ class ContractsListingRequestBuilder {
     const contractIdEncoded = new TextEncoder('utf-8').encode(this.contractId);
 
     const buffer = new Uint8Array(
-        contractIdEncoded.byteLength + certHolderId.byteLength +
-        certVersion.byteLength);
+        contractIdEncoded.byteLength + certVersion.byteLength
```

Comment:
What is this change for ?

Reply:
This changes is reflected due to the incorrect order...The updated version will be the correct one...